includes/customizer-functions.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Register Customizer controls.
+ *
+ * @return void
+ */
+function hello_customizer_register( $wp_customize ) {
+	require get_template_directory() . '/includes/customizer/customizer-action-links.php';
+
+	$wp_customize->add_section(
+		'hello-options',
+		[
+			'title' => esc_html__( 'Header & Footer', 'hello-elementor' ),
+			'capability' => 'edit_theme_options',
+		]
+	);
+
+	$wp_customize->add_setting(
+		'hello-header-footer',
+		[
+			'sanitize_callback' => false,
+			'transport' => 'refresh',
+		]
+	);
+
+	$wp_customize->add_control(
+		new HelloElementor\Includes\Customizer\Hello_Customizer_Action_Links(
+			$wp_customize,
```

Comment:
are we sure it exsists? and loaded?

Reply:
We get the `$wp_customize` parameter from the [customize_register](https://developer.wordpress.org/reference/hooks/customize_register/) hook.

WordPress best practices doesn't require us to check it when hooking to the customizer registration.